### DISCUSSION TITLE
capitalism, assholes, and GPL ( your thoughts on Steam release?)

### DISCUSSION THREAD

**Discussion by Anaglyphic** [CleverRaven/Cataclysm-DDA#49020] [General]
Sorry. You know I have no ability to post tersely. Just ignore me.

Air. Steam. ha. clever.

I immediately want to hate him. Because I know it's got nothing with wanting other people to know about CDDA, it's about him, his ego. his appropriation of something he's legally allowed to. Does he plan to release it with UltiCa like the default tiles compile? Nope of course not, his own tileset will be the default. His own changes to music and sound too, which I'm sure will be curated from other open-licenced things.

But while I'm hating him for doing this and planning an entire shitposting campaign everywhere I can think of, I think of the fact we're on github now owned by Microsoft, who's collecting more metrics off Atom than they do off of VSC _(so I've heard, haven't verified personally)_; and I fully support people stripping Atom of every single telemetry or metric collection thing that phones home, pings repositories, even just a single packet to check for updates or to tally another unique install, without first asking the user's permission.

So how is that any different than what Air is doing? Hmm. I guess I justify it as more "for the people" - not just anti-capitalist, but anti-monetizing-human-existence... where Air et al - it's not like he's the only person doing it, taking open-source code and repackaging it, at least he's not trying to make it commercial immediately. 

For all I know he's actually not a bad guy, he's just motivated to do this. But he clearly has a plan, an outlook, besides just putting it on steam. He's mentioned multiple times about selling modules for the game, campaigns and mods. My assumption being so he won't be hated in the community for "stealing" but can still monetize his efforts, it's a pretty common tactic.

He did say he asked for permission and kevin said no but I can't stop you. Which, credit where due, he didn't have to ask. Air also said he'd transfer it to kevin if he setup a steamworks account, I kind-of want to see what would happen - what he would do if kevin actually did and asked him to transfer it, but then did nothing with it? I'm sure he'd start it right back up. Because this isn't about CDDA being made available to a wider audience on steam, it's about him and his goals.

Still, I hate it and it does depress and de-motivate me that people do this... not specifically about Air. How can anyone participate in a GPL/OSS project when the end result could be used in a way you disagree with? I don't WANT to have that reaction, that emotion. I'd rather participate anyway in spite of what some person does that goes against what the vibe of the project is; but Great Satan it's hard, like forgiving a cheating lover kind of hard.

This isn't an algorithm or library like an image-processing bit that will make it's way into dozens of projects, like dcraw to image related apps, where appropriating the code is expected and even preferred, this is a very specific thing, working towards becoming complete in it's own right, with it's own releases already. And it's active.

Nothing I can do anyway, but bitch about it. :computer: :rage:

**Comment by The-Numbers** [comment]
I don't like the idea of a different steam release managed by a single person either. At the very least I'm afraid of it splitting the player base unnecessarily.

**Reply by faefux** [reply]
I'm confused what is the implication of that exactly? Coming from a place of complete ignorance on such matters: Why does it matter if the player base plays this guys steam one? If is there s ultimately there choice and risks to make and I fail to see how it could affect development.

**Reply by The-Numbers** [reply]
He's making. Few choices already to make it different from the main branch, with his own tile and sound pack. If he decides to make it even more different, in essence it could become a separate game from the main one. That would lead to people playing very similar yet slightly different copies of the same game. The last thing I'd want is for there to be two versions of the exact same game with their own development for absolutely no reason.

**Reply by actual-nh** [reply]
If it's labeled as being the same game, that's especially a problem... EDIT: whether or not a CC-BY-SA 3.0 license requires labeling changes depends on whether it can be considered an "adaptation" or "derivative work", which I find a bit unclear, to put it mildly (BTW, CC-BY-SA 4.0 specifically requires notation of changes). It does definitely state that he can't claim any form of endorsement, even implicitly - it may be possible to convince Steam (if he's unwilling) that he needs to put an explicit disclaimer as to that.

**Reply by ZhilkinSerg** [reply]
> The last thing I'd want is for there to be two versions of the exact same game with their own development for absolutely no reason.

There is nothing wrong in creating a fork (see https://github.com/Whales/Cataclysm, https://github.com/cataclysmbnteam/Cataclysm-BN, https://github.com/zaimoni/Cataclysm and https://github.com/IriaSomobu/cataclysm-nine), but I don't think they are creating one.

**Reply by Fris0uman** [reply]
> his own tile 

Screen shots only show a weirdly edited version of Ultica so far

**Comment by Anaglyphic** [comment]
That we can't "stop" him or whatever wasn't really the issue, and re: being labeled actual-nh he's done more than that, have you seen the edits he made to the biohazard ascii logo scaling it up and making it (and the letters for the splash) radiation-y glow-y?

Even things like that, I'm not saying he's not entitled to spend his free time however he wants, but what if, in the guise of making even a properly-labeled "derivative works", he "give it all away" with his changes, his description, his community (on steam) posts where he'll be flagged as author/creator - just little things like that - imagine if you started your first game knowing about mutations, cybernetics, that 'Zed-Ra' _(intentional misspelling)_ isn't a slutty cousin of She-Ra... for a game of adventure and discovery, where death is a learning experience? What if what he releases is a steam freebie - "play-once, play-twice, bored, delete" scenario? He can literally ruin the game for those that discovered it only via steam in the first place, and will never look elsewhere.

Again, not much we can do about it - we can bitch at him, or try to convince him to adhere to some honourable code he has to come up with for himself on conduct, but... yeah...

**Comment by andrei8l** [comment]
There's a third-party adaptation on the iOS store that also allows for monetization and it somehow hasn't spelt the doom of Cataclysm yet. I think you're over-reacting to one of the liberties of CDDA's license (and free software in general) just because the "wrong" person is taking advantage of it. Would it really be much better if Kevin owned the Steam product? He can certainly be commended for his leadership of this project, but he doesn't own (most of) the code or content either.

Another thing to consider specifically for Steam is that, if Air (or anyone else) tries to sneakily monetize it, his product page will get review-bombed to hell and so he won't get much from it.

**Comment by kevingranade** [comment]
Please restrict discussions to things that concern the project itself, there are quite a lot of places you can vent your spleen about vaguely project-related issues like this.
